.circleci/config.yml

```diff
@@ -18,9 +23,7 @@ jobs:
       - run:
           name: Bundler
           command: |
-            bundle config set --local path '/tmp/bundler'
-            bundle config set --local with 'tests'
-            bundle install
+            bundle install && bundle cache
```

Comment:
Force caching of gems

Reply:
Any reason both commands are run? The [`bundle cache` docs](https://bundler.io/v2.4/man/bundle-cache.1.html#REMOTE-FETCHING) say:
> By default, bundle cache fetches and also installs the gems to the default location.